hbp_knowledge/receptors/wang2016.bel

```diff
@@ -61,27 +61,37 @@ SET Citation = {"PubMed", "26837465"}
 SET Evidence = "In the hippocampal and cortical tissues of all AD cases examined, neuronal staining for ERα  was a prominent finding,\
 both with a weak cytoplasmic stain, but often specifically with a fibrillar appearance suggesting ERα  was in neurofibrillary tangles\
 (Fig. 1A), also shown at a higher magnification (Fig. 1B)."
-SET MeSHAnatomy = {"Cerebral Cortex", "Hippocampus"}
 SET Confidence = "Medium"
+SET MeSHAnatomy = {"Cerebral Cortex", "Hippocampus"}
 p(HGNC:ESR1) -- complex(GO:"neurofibrillary tangle")
+composite(p(GO:"neurofibrillary tangle"),p(HGNC:ESR1)) -> complex(p(GO:"neurofibrillary tangle"),p(HGNC:ESR1))
+#Anatomy was unset. Can above Statement be introduced?
+UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "To confirm the localization of ERα  in NFTs, double fluorescence staining with antibodies to ERα  and hyperphosphorylated\
 tau (PHF-1) revealed that ERα -positive NFTs were also positive for PHF-1 (Fig. 2A–F)."
 SET Confidence = "Medium"
 p(HGNC:ESR1) -- complex(GO:"neurofibrillary tangle")
-p(HGNC:ESR1) -- p(HGNC:MAPT, pmod(HBP:hyperphosphorylation))
+complex(p(GO:"neurofibrillary tangle"),p(HGNC:ESR1),p(HGNC:MAPT, pmod(HBP:hyperphosphorylation)))
```

Comment:
see above comment...
you could add composite of NFTs and ESR1 is associated with NFTs and MAPT
```suggestion
composite(p(GO:"neurofibrillary tangle"), p(HGNC:ESR1)) -- composite(p(GO:"neurofibrillary tangle"), p(HGNC:MAPT, pmod(HBP:hyperphosphorylation)))
```